clang/lib/DPCT/CallExprRewriterWmma.cpp

```diff
@@ -19,6 +21,13 @@ std::function<bool(const CallExpr *)> checkEnableJointMatrix() {
 }
 
 void CallExprRewriterFactoryBase::initRewriterMapWmma() {
+  if (DpctGlobalInfo::useExtJointMatrix()) {
```

Comment:
Add a comment to explain which funciton migration requires the enum and why we cannot init it when construct EnumNamesMap?